src/encrypt.js

```diff
@@ -31,20 +30,29 @@ const MAX_PAYLOAD_LENGTH = 4080;
  * - {@link https://en.wikipedia.org/wiki/Elliptic_curve_Diffie%E2%80%93Hellman}
  * - {@link https://tools.ietf.org/html/draft-ietf-webpush-encryption}
  *
- * @param  {String} message      The message to be sent
- * @param  {Object} subscription The subscription details for the client
- * @return {Object}              An Object containing the encrypted payload and
- *                               the other encryption information needed to send
- *                               the message.
+ * @param  {String} message       The message to be sent
+ * @param  {Object} subscription  The subscription details for the client
+ * @param  {number} paddingLength Number of bytes of padding to use
+ * @return {Object}               An Object containing the encrypted payload and
+ *                                the other encryption information needed to
+ *                                send the message.
  */
-function encrypt(message, subscription) {
+function encrypt(message, subscription, paddingLength) {
+  paddingLength = paddingLength || 0;
+
   // Create Buffers for all of the inputs
-  const plaintext = new Buffer(message, 'utf8');
-  if (plaintext.length > MAX_PAYLOAD_LENGTH) {
+  const padding = makePadding(paddingLength);
```

Comment:
Could you rename this to paddingBuffer.

Could you change paddingLength to force increading the padding length to the max, otherwise use 0. At the moment the proposed API required figuring out of bytelength of message and then figuring out the padding length (which I'm unsure if that would include the 2 bytes or not).